mirdata/datasets/mir_1k.py

```diff
@@ -0,0 +1,323 @@
+
+    1. unvoiced stop
+    2. unvoiced fricative and affricate
+    3. /h/
+    4. inhaling sound
+    5. other (including voiced frames and frames without any vocal activity)
+
+    The dataset was collected by the MIR Lab at National Taiwan University and first published in [1].
+
+    The MIR Lab offers the dataset for download on their website [2] without specifying a license.
+
+    [1] Hsu, Chao-Ling and Jang, Jyh-Shing Roger. "On the Improvement of Singing Voice Separation for Monaural Recordings Using the MIR-1K Dataset." IEEE Transactions on Audio, Speech, and Language Processing (2010)
+
+    [2] http://mirlab.org/dataset/public
+"""
+
+from typing import BinaryIO, Optional, TextIO, Tuple
+
+import librosa
+import numpy as np
+from smart_open import open
+
+from mirdata import annotations, core, download_utils, jams_utils, io
+
+BIBTEX = """
+@article{hsu2010improvement,
+  author={Hsu, Chao-Ling and Jang, Jyh-Shing Roger},
+  journal={IEEE Transactions on Audio, Speech, and Language Processing}, 
+  title={On the Improvement of Singing Voice Separation for Monaural Recordings Using the MIR-1K Dataset}, 
+  year={2010},
+  volume={18},
+  number={2},
+  pages={310-319},
+  doi={10.1109/TASL.2009.2026503}
+}
+"""
+
+INDEXES = {
+    "default": "1.0",
+    "test": "sample",
+    "1.0": core.Index(
+        filename="mir_1k_index_1.0.json",
+        url="https://zenodo.org/records/14428193/files/mir_1k_index_1.0.json?download=1",
+        checksum="c93b0e9145d590f2cff86d02aa2ee855",
+    ),
+    "sample": core.Index(filename="mir_1k_index_1.0_sample.json"),
+}
+
+REMOTES = {
+    "mirlab": download_utils.RemoteFileMetadata(
+        filename="MIR-1K.zip",
+        url="http://mirlab.org/dataset/public/MIR-1K.zip",
+        checksum="3a4e5acd740110ae481a836460dabb6a",
+        unpack_directories=["MIR-1K"],
+    )
+}
+
+LICENSE_INFO = "Unknown"
+
+UNVOICED_LABELS = {
+    "1": "unvoiced stop",
+    "2": "unvoiced fricative and affricate",
+    "3": "/h/",
+    "4": "inhaling sound",
+    "5": "other",
+}
+
+
+class Track(core.Track):
+    """mir_1k Track class
+
+    Args:
+        track_id (str): track id of the track
+
+    Attributes:
+        audio_path (str): path to the track's audio file
+        f0_path (str): path to the track's f0 annotation file
+        lyrics_path (str): path to the track's lyrics file
+        track_id (str): track id
+        unvoiced_labels_path (str): path to the track's unvoiced labels
+        vocal_activity_path (str): path to the track's vocal activity labels
+
+    Cached Properties:
+        f0 (F0Data): the track's f0 annotation
+        lyrics (LyricData): the track's lyrics
```

Comment:
maybe if we end up leaving the lyrics with no interval/duration information, we could also let the user know here?